File: fixtures/adversarial/T2/ts/silent/before/src/format.test.ts
import { format } from "./format";

describe("format", () => {
  it("pads to the width", () => {
    expect(format("a", 3)).toBe("a  ");
    expect(format("ab", 3)).toBe("ab ");
    expect(format("", 3)).toBe("   ");
  });

  it("truncates past the width", () => {
    expect(format("abcd", 3)).toBe("abc");
  });
});

File: fixtures/adversarial/T8/go/silent/after/format_test.go
package format

import "testing"

func TestPadsToTheWidth(t *testing.T) {
	if Format("a", 3) != "a  " {
		t.Fatal("the value should be padded to the width")
	}
}

func TestTruncatesPastTheWidth(t *testing.T) {
	if Format("abcd", 3) != "abc" {
		t.Fatal("the value should be cut at the width")
	}
}
